website/components/navbar.js

```diff
@@ -1,62 +1,58 @@
+import Router from 'next/router'
+import Link from 'next/link'
+import { getMagic } from '../lib/magic.js'
 import Button from './button.js'
+import { useUserContext } from '../lib/user.js'
+
+export default function Navbar({ bgColor = 'nsorange' }) {
+  const [user, setUser] = useUserContext()
+  function logout() {
+    getMagic().user.logout()
+    setUser(undefined)
+    Router.push('/')
+  }
 
-export default function Navbar({ user, loginUrl, bgColor = 'nsorange' }) {
   return (
-    <nav className={`dn dn-m db-ns bg-${bgColor}`}>
+    <nav className={` bg-${bgColor}`}>
```

Comment:
Those classes are there so the navbar disappears on mobile as it did not render well. We need a hamburger or something really. Could you open an issue to get it fixed?

Reply:
yes i will, i have tweak the navbar and its good enough for now in mobile